plenoptic/synthesize/eigendistortion.py

```diff
@@ -3,7 +3,7 @@
 from .autodiff import jacobian, vector_jacobian_product, jacobian_vector_product
 import numpy as np
 import warnings
-from tqdm import tqdm
+from tqdm.auto import tqdm
```

Comment:
Just want to flag this -- if we go with `tqdm.auto`, we'll want to be consistent across all uses of it, and probably add a bit of information about additional requirements involved (at least [ipywidgets](https://ipywidgets.readthedocs.io/en/stable/user_install.html); I've had issues getting it the notebook tqdm working before).

Reply:
I agree w/ you. In running the NBs I noticed the ascii progress bars looked really wonky ( and decided if users were going to run this in Binder then they should look polished.